README.md

```diff
@@ -111,7 +101,15 @@ _**NOTE:** If opting out of one or both of these plugins, **remove** the desired
 
 5. Copy the `.env.sample` into a new `.env` file, and update any necessary `environment` variables in `docker-compose.yml`
```

Comment:
Does the addition of `./bin/theme` replace the need to update `environment` variables in `docker-compose.yml`?

Reply:
yes, good catch!